packages/node_modules/@webex/plugin-meetings/src/roap/index.js

```diff
@@ -220,7 +220,7 @@ export default class Roap extends StatelessWebexPlugin {
    * @memberof Roap
    */
   sendRoapMediaRequest(options) {
-    const {meeting} = options;
+    const {meeting, reconnect} = options;
```

Comment:
Please update the `RoapOptions` typedef on line 12 with this reconnect property.